### DISCUSSION TITLE
No routes matched location "/build/.../*.js"

### DISCUSSION THREAD

**Discussion by ghost** [remix-run/remix#6094] [Q&A]
I noticed that I keep getting messages `No routes matched location "/build/.../*.js"` on terminal whenever I navigate or reload the app that I'm working on so I decided to make a fresh remix app and see if the messages were going to be in there too. This is the result:

```
npx create@remix@latest ./test-remix
Just the basics
Remix App Server
Typescript
npm install - Yes
```

![no routes matched](https://user-images.githubusercontent.com/95581383/232668712-bc81cb8b-4a00-4cfc-9a85-0029da9bc354.gif)

The web app is running how its supposed to be on the browser. Are these messages on the terminal normal?

**Comment by joshpachner** [comment]
Curious, was this ever solved?

**Reply by joshpachner** [reply]
'Cause for us, this error was caused because the build folder wasn't getting populated cause there was an error in the code ( a variable was declared twice within a function)

**Reply by Hillsie** [reply]
Done a fresh install with most current version as of today's date `7 July` and bug happens for me.

 - Remix: `v1.19.0` , 
 - Option: `Just the basics`

**Comment by atalayozmen** [comment]
Was a solution found for this?
